shoop/simple_pricing/module.py

```diff
@@ -29,50 +31,52 @@ def get_context_from_request(self, request):
         customer = getattr(request, "customer", None)
         shop = getattr(request, "shop", None)
 
+        assert shop is not None
+
         if not customer or customer.is_anonymous:
             customer_group_ids = []
         else:
             customer_group_ids = sorted(customer.groups.all().values_list("id", flat=True))
 
         return self.pricing_context_class(
-            shop_id=(shop.pk if shop else None),
+            shop=(shop if shop else None),
```

Comment:
Since `shop` is always required, this ternary operator shouldn't be required anyway.